[BugFix] Fixing descriptions of inversed NamedTransforms

ajymoonILM opened the pull request:
Generate the transform description based on the transform's direction.

Resolves: #111

doug-walker commented:
Thanks for this fix @ajymoonILM !  I had to manually start the CI unit tests, since you are a new contributor.  It looks like they will succeed.

Please let us know if you have suggestions for how to improve the installation instructions (at a minimum it seems we need to clarify that one should start a Poetry shell before using the invoke command).

KelSolaar commented:
Thanks a lot @ajymoonILM!

I haven't checked the artifacts but the code seems to do what we want! The only thing I have to say is that I would prefer to rename `inverse_direction` to `direction` and that we pass a Literal to it, it would be consistent with OCIO and the rest of the current API (and that of OCIO): https://github.com/search?q=repo%3AAcademySoftwareFoundation%2FOpenColorIO-Config-ACES%20direction%3D&type=code

ajymoonILM commented:
Thanks @KelSolaar ! 
Makes sense to me to keep consistency! Just pushed a new commit that addresses your review. Let me know if there's anything else that needs to be updated.

KelSolaar commented on opencolorio_config_aces/config/cg/generate/config.py:
```diff
@@ -204,7 +205,7 @@ def clf_transform_to_description(
         ):
             if clf_transform.description is not None:
-                if inverse_direction:
+                if direction == "Forward":
```
maybe `if direction.lower() == "forward":` so that it becomes case insensitive?

LGTM otherwise and thank you!

ajymoonILM replied:
Updated with the case insensitive check.
Let me know of anything else that needs to be addressed!

KelSolaar replied:
We are good! Thanks a lot @ajymoonILM!

KelSolaar reviewed (approved):
LGTM!
